### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: metadataStore/test/test_commands.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+def teardown():
+    conn = mongoengine.connection.get_connection('test_db')
+    conn.drop_database(db_name)
+    conn.drop_database(dummy_db_name)
+
+
+def context_decorator(func):
+    def inner(*args, **kwargs):
+        with switch_db(BeamlineConfig, 'test_db'), \
```
Comment (ericdill) [inline metadataStore/test/test_commands.py:38]: use `()` to get rid of the `\`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - source activate testenv
+  - pip install coveralls mongoengine
+  - python setup.py install
+
+
+script:
+  - python run_tests.py
+
+after_success:
+  coveralls
```
Comment (ericdill) [inline .travis.yml:30]: hah, you already have the basics of coveralls ready for me

File: metadataStore/test/test_commands.py (hunk 1 above)
Comment (tacaswell) [inline metadataStore/test/test_commands.py:38]: For reasons that I don't recall right now, you can't do that with context managers.  This use case is the only reason `\` line continuation has not been deprecated.

Comment (ericdill) [inline metadataStore/test/test_commands.py:38]: ah well fair enough

**Comment by ericdill** [comment]
looks fine to me

**Comment by tacaswell** [comment]
Holding off on further tests until after tomorrow morning

**Pull request merged**